src/prss.rs

```diff
@@ -471,7 +435,7 @@ pub mod test {
         // There isn't enough entropy in this field to be sure that the test will pass.
         // So run a few rounds looking for a mismatch.
         let mut v2 = Fp31::from(0_u8);
-        for _ in 0..12 {
+        for _ in 0..21 {
```

Comment:
just fyi - I am still seeing sporadic failures so maybe we can make it an infinite loop or restructure the test to make it stable

```bash
thread 'prss::test::three_party_random' panicked at 'assertion failed: `(left != right)`
  left: `11_mod31`,
 right: `11_mod31`', src/prss.rs:448:9
```

Reply:
That's concerning.  Maybe the problem is the birthday paradox is causing this to be a much higher rate of collision.  I had this pegged at 1/31 odds of a collision each run, so (1/31)^21 odds, which is close to 2^-100.  If we get collisions on that, maybe there is a bug.